learning_wavelets/training_scripts/exact_recon_unet_training.py

```diff
@@ -0,0 +1,95 @@
+import os
+import os.path as op
+import time
+
+from tensorflow.keras.callbacks import TensorBoard, ModelCheckpoint, LearningRateScheduler
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from learning_wavelets.config import LOGS_DIR, CHECKPOINTS_DIR
+from tensorflow.keras.optimizers import Adam
+from learning_wavelets.data.datasets import im_dataset_div2k, im_dataset_bsd500
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+tf.random.set_seed(1)
+
+
+def train_unet(noise_std_train, noise_std_val, n_samples, source, cuda_visible_devices, base_n_filters):
+
+    os.environ["CUDA_VISIBLE_DEVICES"] = ','.join(cuda_visible_devices)
```

Comment:
```suggestion
def train_unet(noise_std_train, noise_std_val, n_samples, source, base_n_filters):
```

As I was saying, let's get rid of the cuda stuff